### PR TITLE
keep LeapCSharp up to date with platform 562d155

### PR DESCRIPTION
**Pull request by veronicaz41**
This includes the following PR:
Dispatch Connect and Init events to clients if we have already inited… #5154
Remove phone tracking and dependencies #5252

### CODE REVIEW

File: Assets/LeapC/Connection.cs [hunk 1]
```diff
@@ -253,13 +260,13 @@ private void processMessages()
...
     {
       Frame newFrame = frameFactory.makeFrame(ref trackingMsg);
-      if (_trackedQuadsAreEnabled)
-        newFrame.TrackedQuad = this.findTrackQuadForFrame(newFrame.Id);
       Frames.Put(newFrame);
       this.LeapFrame.Dispatch<FrameEventArgs>(this, new FrameEventArgs(newFrame));
     }
 
-
+    int requestCount = 0;
```
Comment (Amarcolina) [inline Assets/LeapC/Connection.cs]: This variable is never used

**Comment by Amarcolina** [comment]
I am getting errors in the AR scene

Exception: System.NullReferenceException: Object reference not set to an instance of an object
  at Leap.Config.handleConfigChange (System.Object sender, Leap.ConfigChangeEventArgs eventArgs) [0x0001f] in C:\Users\Alex\Documents\GitHub\UnityModules\Assets\LeapC\Config.cs:49 
  at (wrapper delegate-invoke) System.EventHandler`1<Leap.ConfigChangeEventArgs>:invoke_void__this___object_ConfigChangeEventArgs (object,Leap.ConfigChangeEventArgs)
  at (wrapper delegate-invoke) System.EventHandler`1<Leap.ConfigChangeEventArgs>:invoke_void__this___object_ConfigChangeEventArgs (object,Leap.ConfigChangeEventArgs)
  at (wrapper delegate-invoke) System.EventHandler`1<Leap.ConfigChangeEventArgs>:invoke_void__this___object_ConfigChangeEventArgs (object,Leap.ConfigChangeEventArgs)
  at Leap.CSharpExtensions.Dispatch[ConfigChangeEventArgs] (System.EventHandler`1 handler, System.Object sender, Leap.ConfigChangeEventArgs eventArgs) [0x00006] in C:\Users\Alex\Documents\GitHub\UnityModules\Assets\LeapC\CSharpExtensions.cs:79 
  at LeapInternal.Connection.handleConfigChange (LeapInternal.LEAP_CONFIG_CHANGE_EVENT& configEvent) [0x0004b] in C:\Users\Alex\Documents\GitHub\UnityModules\Assets\LeapC\Connection.cs:525 
  at LeapInternal.Connection.processMessages () [0x00206] in C:\Users\Alex\Documents\GitHub\UnityModules\Assets\LeapC\Connection.cs:241 

Both Images and Tracking are not functioning in this scene.

**Comment by Amarcolina** [comment]
Confirmed that I do *not* get errors and *do* get images/tracking in the AR scene on develop

**Comment by veronicaz41** [comment]
The develop is already brought up to date, closing this

**Pull request closed without merging**